src/app/services/endpoints.service.ts

```diff
@@ -208,6 +208,14 @@ export class EndpointsService {
     return `${this.jobs}challenge_phase_split/${phaseSplitId}/leaderboard/?page_size=1000`;
   }
 
+  /**
+   * Challenge Leaderboard fetch
+   * @param phaseSplitId  phase split id
+   */
+  challengeLatestSubmissionURL(phaseSplitId) {
```

Comment:
Can we please change this to `particularChallengePhaseSplitUrl` or something like this?